examples/notebooks/tour-of-beam/getting-started.ipynb

```diff
@@ -0,0 +1,1016 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Getting started -- Tour of Beam",
+      "provenance": [],
+      "toc_visible": true,
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/apache/beam/blob/master/examples/notebooks/tour-of-beam/getting-started.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "SSKEd7tP-b2k",
+        "cellView": "form"
+      },
+      "source": [
+        "#@title Licensed to the Apache Software Foundation (ASF), Version 2.0 (the \"License\")\n",
```

Comment:
This is showing as a large cell at the top? Do we need this as a large title there?

Reply:
Apparently we can make this an H6 and it appears smaller, I didn't know a `#@title` line was also Markdown.